src/sentry/api/endpoints/organization_teams.py

```diff
@@ -48,15 +48,25 @@ class OrganizationTeamsPermission(OrganizationPermission):
     }
 
 
+@extend_schema_serializer(exclude_fields=["idp_provisioned"])
 class TeamPostSerializer(serializers.Serializer, PreventNumericSlugMixin):
-    name = serializers.CharField(max_length=64, required=False, allow_null=True, allow_blank=True)
     slug = serializers.RegexField(
         DEFAULT_SLUG_PATTERN,
+        help_text="""Uniquely identifies a team and is used for the interface. If not
+        provided, it is automatically generated from the name.""",
         max_length=50,
         required=False,
         allow_null=True,
         error_messages={"invalid": DEFAULT_SLUG_ERROR_MESSAGE},
     )
+    name = serializers.CharField(
+        help_text="""A deprecated field that can be used for an external customer
+        ID if needed. If not provided, it is automatically generated from the slug.""",
```

Comment:
what is external customer ID? maybe we can say it can be used as a human friendly title for the team?

Reply:
Discussed w/ @JoshFerge. We can't remove `name` from the docs b/c its' still returned in the response, but marking it as deprecated in the docs and request schema is good enough.